api/utils/filters.py

```diff
@@ -32,29 +32,33 @@ def get_time_filtered_temperatura(request):
 
 
 def get_time_filtered_pauta(request):
+    '''
+    Filtra as pautas que ocorreram depois ou no dia da data_referencia. Se a
+    data_referencia é uma sexta-feira, retornará todas as pautas com um
+    período de 6 dias após, se não,retorna as pautas da mesma semana da data_referencia.
+    '''
     data_referencia = request.query_params.get('data_referencia')
-    semanas_anteriores = request.query_params.get('semanas_anteriores')
 
     queryset = PautaHistorico.objects
-
     date = None
+
     if data_referencia:
         try:
             date = datetime.strptime(data_referencia, '%Y-%m-%d')
         except ValueError:
             print(
-                f'Data de referência ({data_referencia}) inválida. '
-                'Utilizando data atual como data de referência.')
+                f'Data de referência ({data_referencia}) inválida. ')
+            date = datetime.datetime.now().strptime('%Y-%m-%d')
```

Comment:
Acho que essa parte está sobrando, né? `.strptime('%Y-%m-%d')`
O `now()` já é uma data, `strptime` é para criar uma data a partir de uma string.

Reply:
Minha intenção era formatar a data pra esse formato `
%Y-%m-%d`, acho que exista outra forma de fazer isso :thinking:

Reply:
Mas `date` precisa ser um objeto datetime, para depois fazer as operações que estão mais abaixo na função, não? Objetos datetime não têm formato, o formato é usado para depois gerar uma string a partir deles.

Reply:
É verdade, acabei me confundindo. De qualquer forma tirei de colocar a data de hoje, eu deveria deixar?

Reply:
Estou meio perdido... Qual o papel desse endpoint?
Pelas suas últimas alterações parece que ficou sendo: retornar se está em pauta nos próximos dias após a data de referência. É isso?
Acho que isso é o que precisamos no frontend no momento, certo?

O que o endpoint deixou de ter é a capacidade de retornar se estava em pauta em cada uma das últimas X semanas. Pq agora não tem mais o `semanas_anteriores`, né? Mas é algo que não estamos usando no momento, né?

Reply:
Ele sempre vai retornar as pautas da semana de acordo com a data de referência, se for uma sexta, ele retorna a da próxima semana. Estamos seguindo essa lógica por causa do front.


O front não perdeu a funcionalidade de ver pautas antigas, continua da mesma forma, essa PR é só para resolver o bug de retornar pautas que não são da semana da data de referência

Reply:
Roda essa branch em conjunto com o front, existe uma pauta no dia 25 desse mês. Pode visualizar como se comporta

Reply:
Não falei que o front perdeu funcionalidade. Pelo contrário, o endpoint está na medida do que o front precisa.
O que disse é que o endpoint perdeu a capacidade de retornar a lista de datas em que a proposição estava em pauta em um período maior. Por exemplo, em quais semanas de 2018 essa proposição esteve em pauta?
Mas é algo que não estamos usando no front agora.
É uma escolha entre deixar o endpoint mais focado nas nossas necessidades de agora, ou mais genérico.

Reply:
Bem, falaram que ele estar retornando antigas era um problema, deu até um problema na entrega e Tarciso teve que editar os csv na mão pra não deixar retornar pautas antigas

Reply:
Entendi. Vou revisar o PR de Jair e depois testo e aceito esse então.